bin/update

```diff
@@ -28,18 +40,20 @@ chdir APP_ROOT do
   # If we need a "reset all the things" script, we should call it bin/reset.
   system! 'bundle update'
 
+  # load 'Rakefile'
+  load_rakefile
   puts "\n== Updating database =="
-  system! 'bin/rails db:migrate'
+  Rake::Task["db:migrate"].invoke
 
   puts "\n== Seeding database =="
-  system! "bin/rails db:seed GOOD_MIGRATIONS=skip"
+  Rake::Task["db:seed"].invoke
 
   puts "\n== Resetting tests =="
   system! "bin/rails test:vmdb:setup"
```

Comment:
> To be honest if the PR description is longer than 2 sentences, I rarely read it.

@Fryguy The reason I write such lengthy PR descriptions is rarely for myself (though, it is nice to reflect on the process to getting to a conclusion to self-validate before asking for peer review), but for the reviewer to avoid such questions like this that are sometimes already known by requester ahead of time.  This also allows the reviewer to get caught up to speed quicker, avoid of asking questions with known answers, context switching, and then coming back to an answer and having to re-familiarize themselves with the changes.  Clearly, though, this is an ineffective strategy on my part.

That all said, I find that have to answer comments from reviewer(s) ineffective in most cases for describing the authors original intent and makes looking back on a PR in a historical context that much harder to grok.

> I wasn't sure if that means the same thing as what I'm referring to above.

@jrafanie Effectively yes.  While the issue the you brought up was between `db:migrate` and `db:seed`, which act on the same database (`development` v.s. `production` v.s. `test`), where as `test:vmdb:setup` makes a connection to the `test` database in order to drop and re-create it.

This duplicate connection issue was what I was referring to, though I could see this happening in other places as well not knowing the full extent all of Rails' rake tasks and associated prereq, hence the emphasized consideration.